.eslintrc.js

```diff
@@ -31,7 +31,9 @@ module.exports = {
       "plugins": [
         "@typescript-eslint"
       ],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-explicit-any":0
```

Comment:
In this and the other PR, I wonder if we should just disable the linter for the particular lines that use `any` legitimately. I think we want to generally discourage its use.

@kevcenteno, thoughts?

Reply:
I agree 100%.